src/main/java/com/training/pojos/ga/createGa/TermsAndConditionsAgreement.java

```diff
@@ -0,0 +1,9 @@
+package com.training.pojos.ga.createGa;
```

Comment:
Please follow the correct naming convention for package: 
https://www.geeksforgeeks.org/java-naming-conventions/